package.json

```diff
@@ -75,6 +75,11 @@
     ]
   },
   "homepage": "https://github.com/department-of-veterans-affairs/vets-website#readme",
+  "browserslist": [
+    "last 1 version",
+    "> 1%",
+    "ie 11"
+  ],
```

Comment:
[Autoprefixer uses Browserslist](https://github.com/postcss/autoprefixer#browsers)

Reply:
I think we should set this to more closely match the browsers we support, which are listed in babelrc:

`["Chrome 60", "Firefox 57", "iOS 9", "Edge 14", "ChromeAndroid 64", "Safari 10", "ie 11"]`

Reply:
We can probably just remove the browser list from babelrc if this is present. https://github.com/browserslist/browserslist-example#babel

Reply:
Not quite. The actual active browser list in babelrc is just modern browsers. The full supported list that I pasted above is commented out, but that's what we'd want to use for autoprefixer. We have the two different lists in babel so that we can bundle polyfills for older browsers separately.

Reply:
It seems that Firefox 57 isn't part of the `caniuse-db` module used under the hood - `Module build failed: BrowserslistError: Unknown version 57 of Firefox`. I changed that to FireFox 56, and it worked though.

I confirmed by checking `brandConsolidation.css` for instances of `webkit-appearance`. This suggestion and the rules before it both output `select{-webkit-appearance:none;-moz-appearance:none;appearance:none;`, while the current Staging CSS does not have that.

Reply:
Pushed the update ✔️